src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -41,6 +44,13 @@ public CorInfoImpl(Compilation compilation)
 
             _comp = CreateUnmanagedInstance();
 
+            if (!IsWindows())
```

Comment:
(Note that we eventually want to self-host this toolchain on CoreRT, but we do not plan to build the managed-native exception for CoreRT - even on Windows.)